src/components/Library/Dropdown.js

```diff
@@ -0,0 +1,25 @@
+import React, { useState } from 'react';
+
+export default function Dropdown() {
+  return(
+    <div className="dropdown is-hoverable">
+
+      <div className="dropdown-trigger">
+          <span className="icon is-small">
+            <i className="fas fa-angle-down" aria-hidden="true"></i>
+          </span>
+
+      </div>
+      <div className="dropdown-menu" id="dropdown-menu" role="menu">
+        <div className="dropdown-content">
+          <a href="#" className="dropdown-item">
+            Edit
+          </a>
+          <a className="dropdown-item">
+            Delete
+          </a>
```

Comment:
It is not correct to put "Edit" and "Delete" in the component directly. This makes your component completely non-reusable, while the idea of React is to build small reusable components.

In React, to make components reusable, we use props. Props allow to pass context specific values to a "dumb" component, meaning it does not know anything of the context in which it is used.

In this case, you have two solutions:

1°) The first one is to pass a list of items which describe the items that the dropdown should display, and other specificities such as what should happen when the item is clicked, what colour it should have, ...
```javascript
items={[{
  label: 'Edit',
  onClick: onEdit // Replace with what makes sense in your case
}, {
  label: 'Delete',
  onClick: onDelete // Replace with what makes sense in your case
}
]}
```

2°) 
Create another component called `DropdownItem` which you must import too from the parent component. This DropdownItem component is very simple and would be used like so:
```jsx
<DropdownItem onClick={onEdit}>Edit</DropdownItem>
```

Which you would then insert inside your Dropdown component (using the prop called "children", check React doc to better understand this) like so:
```jsx
<Dropdown>
  <DropdownItem onClick={onEdit}>Edit</DropdownItem>
  <DropdownItem onClick={onDelete}>Delete</DropdownItem>
</Dropdown>
```
You can see the `Dropdown` and `DropdownItem` components as simply wrappers around Bulma classes to abstract them away. If one day you want to use another CSS lib as Bulma, you would just need to change the classes in all the small components, but your main code would not change at all.